boards/arm/xmc45_relax_kit/xmc45_relax_kit.dts

```diff
@@ -27,13 +27,6 @@
 	clock-frequency = <120000000>;
 };
 
-&usic1ch0 {
```

Comment:
IMO it makes sense to disable the node here without renaming the soc dtsi

Reply:
Hi @parthitce, thanks for reviewing.
The usic1ch0 node is already initially disabled in xmc4xxx.dtsi. Do you want to keep the settings and just set it as disabled?

Reply:
> IMO it makes sense to disable the node here without renaming the soc dtsi

Are you against renaming the file from xmc4500.dtsi to xmc4500_F100x1024.dtsi? Different versions of the XMC4500_xx have different flash and sram sizes. I don't see a clean way to have this without different xmc4500_AxB.dtsi files for each one.
![Screenshot_2022-07-03_09-19-44](https://user-images.githubusercontent.com/17433307/177042409-b546a3d2-d210-4561-b794-c10b3652b283.png)